### PR TITLE
build(docker): Fix version details in docker image

### LINKED ISSUE #1468: Broken `version` and `--version` with v1.32.0
Thank you for creating the issue!

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of golangci-lint</summary>

v1.32.0

</details>

<details><summary>Config file</summary>

None.

</details>

<details><summary>Go environment</summary>

None. (Docker)

</details>

<details open><summary>Verbose output of running</summary>

**v1.31.0**:

```console
$ docker run --rm golangci/golangci-lint:v1.31.0 golangci-lint version
golangci-lint has version 1.31.0 built from 3d6d0e7 on 2020-09-07T15:14:41Z

$ docker run --rm golangci/golangci-lint:v1.31.0 golangci-lint --version
golangci-lint has version 1.31.0 built from 3d6d0e7 on 2020-09-07T15:14:41Z
```

**v1.32.0**:

```console
$ docker run --rm golangci/golangci-lint:v1.32.0 golangci-lint version
golangci-lint has version master built from ? on

$ docker run --rm golangci/golangci-lint:v1.32.0 golangci-lint --version
Error: unknown flag: --version
Usage:
  golangci-lint [flags]
  golangci-lint [command]

Available Commands:
  cache       Cache control and information
  completion  Output completion script
  config      Config
  help        Help
  linters     List current linters configuration
  run         Run this tool in cloud on every github pull request in https://golangci.com for free (public repos)
  version     Version

Flags:
      --color string              Use color when printing; can be 'always', 'auto', or 'never' (default "auto")
  -j, --concurrency int           Concurrency (default NumCPU) (default 8)
      --cpu-profile-path string   Path to CPU profile output file
  -h, --help                      help for golangci-lint
      --mem-profile-path string   Path to memory profile output file
      --trace-path string         Path to trace output file
  -v, --verbose                   verbose output

Use "golangci-lint [command] --help" for more information about a command.

failed executing command with error unknown flag: --version
```

</details>

I noticed that `golangci-lint version` and `golangci-lint --version` are broken with v1.32.0:

- `version` outputs invalid text: `golangci-lint has version master built from ? on`
- `--version` is handled as an unknown flag: `Error: unknown flag: --version`

### PR DESCRIPTION
**Pull request by sayboras**
## Description

As part of #1383, multi-arch docker build was supported. However,
ldflags for version details and CGO_ENABLED=0 were missing.

This commit is to add -ldflags as part of Docker build.

Fixes #1468

Signed-off-by: Tam Mach <sayboras@yahoo.com>

## Testing

Testing was done with my docker hub and tag v1.32.1

<details>
<summary>Testing with arm64</summary>

```shell script
$ docker run --rm sayboras/golangci-lint:v1.32.1 golangci-lint version
golangci-lint has version v1.32.1 built from 89647eba on 2020-10-26T10:16:00Z

$ docker run --rm sayboras/golangci-lint:v1.32.1 golangci-lint --version 
golangci-lint has version v1.32.1 built from 89647eba on 2020-10-26T10:16:00Z


$ docker run --rm -w /src -v $PWD:/src sayboras/golangci-lint:v1.32.1 golangci-lint run ./...

$ uname -a
Linux ubuntu 5.4.0-1021-raspi #24-Ubuntu SMP PREEMPT Mon Oct 5 09:59:23 UTC 2020 aarch64 aarch64 aarch64 GNU/Linux
```

</details>

<details>
<summary>Testing with amd64</summary>

```shell script
$ docker run --rm sayboras/golangci-lint:v1.32.1 golangci-lint version
golangci-lint has version v1.32.1 built from 89647eba on 2020-10-26T10:16:00Z

$ docker run --rm sayboras/golangci-lint:v1.32.1 golangci-lint --version 
golangci-lint has version v1.32.1 built from 89647eba on 2020-10-26T10:16:00Z

$ docker run --rm -w /src -v $PWD:/src sayboras/golangci-lint:v1.32.1 golangci-lint run ./... -v
level=info msg="[config_reader] Config search paths: [./ /src / /root]"
level=info msg="[lintersdb] Active 10 linters: [deadcode errcheck gosimple govet ineffassign staticcheck structcheck typecheck unused varcheck]"
level=info msg="[loader] Go packages loading at mode 575 (imports|name|compiled_files|exports_file|files|deps|types_sizes) took 4.834417647s"
level=info msg="[runner/filename_unadjuster] Pre-built 0 adjustments in 9.205305ms"
level=info msg="[linters context/goanalysis] analyzers took 5.778376571s with top 10 stages: buildir: 4.313237995s, fact_purity: 163.142737ms, inspect: 154.257723ms, ctrlflow: 143.652437ms, printf: 138.532273ms, fact_deprecated: 97.744815ms, ineffassign: 85.125717ms, isgenerated: 50.530356ms, deadcode: 41.686738ms, S1038: 22.20593ms"
level=info msg="[linters context/goanalysis] analyzers took 960.62409ms with top 10 stages: buildir: 837.647488ms, U1000: 122.976602ms"
level=info msg="[runner] Issues before processing: 29, after processing: 0"
level=info msg="[runner] Processors filtering stat (out/in): skip_dirs: 29/29, autogenerated_exclude: 29/29, identifier_marker: 29/29, filename_unadjuster: 29/29, skip_files: 29/29, path_prettifier: 29/29, exclude: 0/29, cgo: 29/29"
level=info msg="[runner] processing took 4.160422ms with stages: exclude: 1.744207ms, identifier_marker: 939.523µs, autogenerated_exclude: 596.772µs, path_prettifier: 532.22µs, skip_dirs: 310.208µs, cgo: 19.848µs, filename_unadjuster: 11.893µs, nolint: 1.222µs, max_same_issues: 923ns, severity-rules: 731ns, uniq_by_line: 411ns, diff: 401ns, skip_files: 370ns, source_code: 312ns, exclude-rules: 290ns, max_from_linter: 260ns, path_shortener: 240ns, path_prefixer: 230ns, max_per_file_from_linter: 191ns, sort_results: 170ns"
level=info msg="[runner] linters took 2.031282027s with stages: goanalysis_metalinter: 1.81668173s, unused: 210.319902ms"
level=info msg="File cache stats: 0 entries of total size 0B"
level=info msg="Memory: 70 samples, avg is 127.8MB, max is 409.7MB"
level=info msg="Execution took 6.88345463s"

$ uname -a                      
Linux ubuntu20 5.4.0-52-generic #57-Ubuntu SMP Thu Oct 15 10:57:00 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

</details>

### CODE REVIEW

**Review by bombsimon — APPROVED** [review]
🎉

**Comment by sayboras** [comment]
Thanks for your review. Once this one is merged, I will release v1.32.1 as well.

Waiting for some more time if others have any feedback.

**Comment by sayboras** [comment]
Seems like there is no other comment, I will merge now :)

**Pull request merged**
